File: test/aidbox_sdk/snapshots/dotnet/hl7-fhir-r4-core/EnrollmentResponse.cs
using Aidbox.FHIR.Base;
using Aidbox.FHIR.Utils;

namespace Aidbox.FHIR.R4.Core;

public class EnrollmentResponse : DomainResource, IResource
{
    public string? Status { get; set; }
    public string? Created { get; set; }
    public string? Outcome { get; set; }
    public Base.ResourceReference? Request { get; set; }
    public Base.Identifier[]? Identifier { get; set; }
    public string? Disposition { get; set; }
    public Base.ResourceReference? Organization { get; set; }
    public Base.ResourceReference? RequestProvider { get; set; }
}

File: test/aidbox_sdk/snapshots/python/hl7_fhir_r4_core/EnrollmentResponse.py
from typing import Optional, List
from dataclasses import dataclass, field
from base import DomainResource
from base import Reference
from base import Identifier

@dataclass(kw_only=True)
class EnrollmentResponse(DomainResource):
    status: Optional[str] = None
    created: Optional[str] = None
    outcome: Optional[str] = None
    request: Optional[Reference] = None
    identifier: Optional[List[Identifier]] = field(default_factory=list)
    disposition: Optional[str] = None
    organization: Optional[Reference] = None
    request_provider: Optional[Reference] = None